gui/mainwindow.cpp

```diff
@@ -470,6 +472,15 @@ void MainWindow::loadSettings()
     }
 }
 
+static ReportType getMisraCReportType(const QStringList &standards)
+{
+    if (standards.contains(CODING_STANDARD_MISRA_C_2023))
+        return ReportType::misraC2023;
+    if (standards.contains(CODING_STANDARD_MISRA_C_2025))
+        return ReportType::misraC2025;
+    return ReportType::misraC2012;
```

Comment:
We can try to fix this later so that Misra C 2012 is added to codingStandards also. But you don't have to do that now in this PR.